gnd/src/main/java/com/google/android/gnd/GndApplicationModule.java

```diff
@@ -120,6 +122,13 @@ static FirebaseFirestore firebaseFirestore(FirebaseFirestoreSettings settings) {
     return firestore;
   }
 
+  /** Returns a reference to the default Storage bucket. */
+  @Provides
+  @Singleton
+  static StorageReference baseFirestoreReference() {
```

Comment:
Firestore != Firebase Storage; please rename to something more appropriate, e.g. `firebaseStorageReference`.